README.md

```diff
@@ -27,98 +25,98 @@ Inside each `.json` file is a single array containing objects. It's easiest to
 illustrate the structure of these with an example:
 
 ```json
-    {
-        "description": "the description of the test case",
-        "schema": {"the schema that should" : "be validated against"},
-        "tests": [
-            {
-                "description": "a specific test of a valid instance",
-                "data": "the instance",
-                "valid": true
-            },
-            {
-                "description": "another specific test this time, invalid",
-                "data": 15,
-                "valid": false
-            }
-        ]
-    }
+{
+    "description": "The description of the test case",
+    "schema": {
+        "description": "The schema against which the data in each test is validated",
+        "type": "string"
+    },
+    "tests": [
+        {
+            "description": "Test for a valid instance",
+            "data": "the instance to validate",
+            "valid": true
+        },
+        {
+            "description": "Test for an invalid instance",
+            "data": 15,
+            "valid": false
+        }
+    ]
+}
 ```
 
-In short: a description, a schema under test, and some tests, where each
-test in the `tests` array is an objects with a description of the case
-itself, the instance under test, and a boolean indicating whether it
-should be valid or invalid.
+In short: a description, a schema under test, and some tests, where each test
+in the `tests` array is an objects with a description of the case itself, the
+instance under test, and a boolean indicating whether it should be valid
+or invalid.
 
-Test Subdirectories
--------------------
+## Test Subdirectories
 
 There is currently only one subdirectory that may exist within each draft
 directory. This is:
 
 1. `optional/`: Contains tests that are considered optional.
 
-Coverage
---------
+## Coverage
 
-Drafts 07, 06, 04 and 03 should have full coverage, with drafts 06 and
-07 being considered current and actively supported.
+Drafts 07, 06, 04 and 03 should have full coverage, with drafts 06 and 07 being
```

Comment:
I think we can consider 2019-09 fully covered now, or very nearly (with the exception of $recursiveAnchor/$recursiveRef).  At the least, we want new tests to be targeting 2019-09 primarily, with similar coverage back to the other drafts when the keywords are similar.  06 is definitely not "current" anymore.

Reply:
I'd prefer to do these text changes in a future PR, with more substantial changes. Those are upcoming, as soon as this gets in.